tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -137,8 +221,15 @@ def test_multiple_pvc_creation_measurement_performance(self, teardown_factory):
         Returns:
 
         """
-        number_of_pvcs = 120
-        log.info("Start creating new 120 PVCs")
+        number_of_pvcs = 12
+        log.info(f"Start creating new {number_of_pvcs} PVCs")
```

Comment:
Do we have validation after PVC's deleted ?

Reply:
currently no, need to be added after this PR and #3355 will be merge